bill/line.go

```diff
@@ -35,16 +32,25 @@ type Line struct {
 	Cost cbc.Code `json:"cost,omitempty" jsonschema:"title=Cost Reference"`
 	// Details about the item, service or good, that is being sold
 	Item *org.Item `json:"item" jsonschema:"title=Item"`
+	// Breakdown of the line item for more detailed information. The sum of all lines
+	// will be used for the item price.
+	Breakdown []*SubLine `json:"breakdown,omitempty" jsonschema:"title=Breakdown"`
 	// Result of quantity multiplied by the item's price (calculated)
-	Sum num.Amount `json:"sum" jsonschema:"title=Sum" jsonschema_extras:"calculated=true"`
+	Sum *num.Amount `json:"sum,omitempty" jsonschema:"title=Sum" jsonschema_extras:"calculated=true"`
```

Comment:
Using pointers will require changes to current implementations of convertors, but is backwards compatible.